<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>575</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="lbeTitle">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>标 题</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="btnMinimize">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
          </property>
          <property name="text">
           <string>_</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnClose">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 11pt &quot;Arial Black&quot;;
color:rgb(255, 55, 55);</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>65</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>选择目标DLL      &gt;&gt;支持直接拖放&lt;&lt;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="edtDllPath">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenFileDiag">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>注入选择</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>选择目标进程:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFresh">
           <property name="text">
            <string>刷 新</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="lstProcess">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>选择注入方式:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbInjectWay">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>远程线程注入</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnLoad">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>开 始 注 入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUnload">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>卸 载 注 入</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>运行日志</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="tbrRunLog"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
